### PR TITLE
add newrelic agent

### PR DESCRIPTION
**Pull request by rydogs**
Using manual method in https://devcenter.heroku.com/articles/deploying-java

Add new relic for monitoring.  To access this, go to https://dashboard.heroku.com/apps/playpingpong/resources and click on New Relic under "Add-ons".  This is a free tier so it may take up to 5 minutes for events to show up

### CODE REVIEW

File: newrelic/newrelic.yml [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+# This section is for settings common to all environments.
+# Do not add anything above this next line.
+common: &default_settings
+
+  # ============================== LICENSE KEY ===============================
+  # You must specify the license key associated with your New Relic
+  # account. For example, if your license key is 12345 use this:
+  # license_key: '12345'
+  # The key binds your Agent's data to your account in the New Relic service.
+  license_key: '9f76c3b721b65bcb9934c8a1ad3e54d01b1d1a0d'
```
Comment (dfurstein) [inline newrelic/newrelic.yml:16]: Not sure there is any way around this, but do we need to have the key be publicly exposed in this file? Would it be possible to make this an environment variable we read in instead?

Comment (rliuAsapp) [inline newrelic/newrelic.yml]: Key is set in heroku,which will overwrite the default.  Already send you both an invite

Comment (rydogs) [reply newrelic/newrelic.yml:16]: This is fine.  Heroku can pass jvm options to overwrite

**Review by gwang16 — APPROVED** [review]

**Review by gwang16 — APPROVED** [review]
Yes boss

**Pull request merged**